test/e2e/pods.go

```diff
@@ -35,19 +35,10 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-// createNamespaceIfDoesNotExist ensures that the namespace with specified name exists, or returns an error
-func createNamespaceIfDoesNotExist(c *client.Client, name string) (*api.Namespace, error) {
-	namespace, err := c.Namespaces().Get(name)
-	if err != nil {
-		namespace, err = c.Namespaces().Create(&api.Namespace{ObjectMeta: api.ObjectMeta{Name: name}})
-	}
-	return namespace, err
-}
-
 func runLivenessTest(c *client.Client, podDescr *api.Pod, expectRestart bool) {
-	ns := "e2e-test-" + string(util.NewUUID())
-	_, err := createNamespaceIfDoesNotExist(c, ns)
-	expectNoError(err, fmt.Sprintf("creating namespace %s", ns))
+	namespace, err := createTestingNS("pods-liveness", c)
```

Comment:
lots of things use `createTestingNS` directly (despite the doc warning)... this seemed to be the most equivalent call... I can switch to `NewFramework` if you or @lavalamp deems it necessary